app/representers/api/series_representer.rb

```diff
@@ -9,9 +9,14 @@ class Api::SeriesRepresenter < Api::BaseRepresenter
   property :created_at, writeable: false
   property :updated_at, writeable: false
   property :app_version, writeable: false
+  property :calendar_ics_url, writeable: false, exec_context: :decorator
```

Comment:
Shouldn't this be a `_link`, not a property?

Reply:
The gist is that you can't "follow" this link (e.g. using the PRXAccess gem), you can't inspect its attributes etc.. It's not a HAL doc. So it seems to more closely follow as an attribute of the model.

Reply:
Hmm... that feels like a shortcoming of the gem.  You should be able to get the URL for a link without following it.  How do we currently handle non-json `type`d links, like images?

```json
{
  "prx:image": {
    "id": 8457,
    "filename": "hermoney-3000x3000-768x768.jpg",
    "size": 109018,
    "_links": {
      "self": {
        "href": "/api/v1/accounts/217417/image",
        "profile": "http://meta.prx.org/model/image/account"
      },
      "profile": {
        "href": "http://meta.prx.org/model/image/account"
      },
      "enclosure": {
        "href": "https://cms.prx.org/pub/1c4c1a73d04f60aaac090731ac1c0c65/0/web/account_image/8457/medium/hermoney-3000x3000-768x768.jpg",
        "type": "image/jpeg"
      },
      "original": {
        "href": "https://cms.prx.org/pub/b56e9d6e928725337d987fa5c3d673f1/0/web/account_image/8457/original/hermoney-3000x3000-768x768.jpg",
        "type": "image/jpeg"
      }
    }
  }
}
```

Reply:
Or maybe I misunderstood your comment.  But in any case, having a non-HAL mime in the link `type` is valid per https://tools.ietf.org/html/draft-kelly-json-hal-06#section-5.3